### DISCUSSION TITLE
write activity

### DISCUSSION THREAD

**Discussion by Pacerier** [community/community#60950] [Projects and Issues]
I need a list of all write activity (issues) on GH of a particular user (eg myself).

Where is the list? (including all revisions)

How to filter it? (eg by search terms / year / repo / etc)

---
<sup>&bull; [misc](https://github.com/orgs/community/discussions/60954)</sup>

**Comment by lshigami** [comment]
Here are the steps on how to find the list of all write activity (issues) on GitHub for a particular user:

Go to the user's profile page.
Click on the Contributions tab.
Use the search bar to filter the list of issues.
Here are some examples of how to filter the list of issues:

To find all issues that the user created, you would search for is:issue author:username.
To find all issues that the user commented on, you would search for is:issue comment:username.
To find all issues that the user closed, you would search for is:issue closed author:username.
To find all issues that the user opened in a specific repo, you would search for is:issue repo:username/reponame.
You can also use the Advanced search options to filter the list of issues even further. For example, you can specify the date range of the issues, the type of issue, or the status of the issue.

**Reply by Pacerier** [reply]
This does not search through revisions.
